datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -307,73 +357,73 @@ SELECT median(c12) FROM aggregate_test_100
 
 # median_i8
 query I
-SELECT median(a) FROM median_i8
+SELECT median(col_i8) FROM median_table
 ----
 -14
 
 # median_i16
 query I
-SELECT median(a) FROM median_i16
+SELECT median(col_i16) FROM median_table
 ----
 -16334
 
 # median_i32
 query I
-SELECT median(a) FROM median_i32
+SELECT median(col_i32) FROM median_table
 ----
 -1073741774
 
 # median_i64
 query I
-SELECT median(a) FROM median_i64
+SELECT median(col_i64) FROM median_table
 ----
 -4611686018427387854
 
 # median_u8
 query I
-SELECT median(a) FROM median_u8
+SELECT median(col_u8) FROM median_table
 ----
 50
 
 # median_u16
 query I
-SELECT median(a) FROM median_u16
+SELECT median(col_u16) FROM median_table
 ----
 50
 
 # median_u32
 query I
-SELECT median(a) FROM median_u32
+SELECT median(col_u32) FROM median_table
 ----
 50
 
 # median_u64
 query I
-SELECT median(a) FROM median_u64
+SELECT median(col_u64) FROM median_table
 ----
 50
 
 # median_f32
 query R
-SELECT median(a) FROM median_f32
+SELECT median(col_f32) FROM median_table
 ----
-3.3
+2.75
```

Comment:
Change `2.75` to `3.3`
Cause by testdata has been changed from 
`1.1, 4.4, 5.5, 3.3, 2.2`
to 
`1.1, 4.4, 3.3, 2.2`
It's makes it easier to create all  the testdata in one table, because other testdata only have 4 column.

Reply:
I agree this makes sense -- thank you for the explanation.